packages/blade/src/components/Counter/Counter.tsx

```diff
@@ -0,0 +1,31 @@
+import { Badge } from '../Badge';
+import type { Feedback } from '~tokens/theme/theme';
+
+export type CounterProps = {
+  value: number;
+  max?: number;
+  intent?: Feedback;
+  contrast?: 'high' | 'low';
+  size?: 'small' | 'medium';
```

Comment:
We can add some JSDoc hints for types

Reply:
Oh added that, forgot to push the commit 🤦